### DISCUSSION TITLE
Support sourceUrl for github packages

### DISCUSSION THREAD

**Discussion by ItsVeryWindy** [renovatebot/renovate#24843] [Suggest an Idea]
### What would you like Renovate to be able to do?

We're using github packages, and I can't get the sourceUrl matching stuff to work. It seems that if I could get it working I could group packages based on that which would be more convenient than doing it manually, and also the packages would link etc.

### If you have any ideas on how this should be implemented, please tell us here.

I've managed to track down the issue to the fact that github packages doesn't seem to support the v3 endpoint to fetch the nuspec file and always returns not found.

https://github.com/renovatebot/renovate/blob/main/lib/modules/datasource/nuget/v3.ts#L203

I do see that in some scenarios it'll set the sourceUrl to the homepage if not set.

It feels logical that maybe that should always do that regardless.

### Is this a feature you are interested in implementing yourself?

Maybe

**Comment by NatMarchand** [comment]
Hello there,
I'm encountering the same issue.
Is there anything I can do?
